### PR TITLE
Register bug

### LINKED ISSUE #106: New User Registration Error
New users are not able to access any pages due to a quota not being set for them upon registering.

### PR DESCRIPTION
**Pull request by riztaha**
Fixed error where new registered users were not able to access webapp due to time_allotment being null.

Added server route logic to set the default quota once a user had registered.

Fixed error in variable declaration.

### CODE REVIEW

File: server/routes/userRoutes.js [hunk 1]
```diff
@@ -60,13 +60,13 @@ module.exports = (db) => {
...
           return res.status(400).send("There was an issue registering.");
         }
-        console.log("successful registration");
         req.session.userId = user.id;
-        res.status(200).send("User created!");
-      })
-      .then((user) => {
-        dbHelper.getUserWithID(req.session.userId).then((user) => {});
+        // console.log("successful user set");
+        dbHelper.addQuotaForUser(user.id, "3 hours");
```
Comment (jpqy) [inline server/routes/userRoutes.js]: This needs a `.then` that runs line 67, and a `.catch` returning status 500 and error.

Remember to do `res.json` not `.send`.

File: server/db/schema/05_quotas.sql [hunk 2]
```diff
@@ -5,7 +5,7 @@ CREATE TABLE quotas
   id                SERIAL    PRIMARY KEY,
 
   user_id           INTEGER   REFERENCES users(id) ON DELETE CASCADE,
-  time_allotment    INTERVAL  NOT NULL default '2 hours',
+  time_allotment    INTERVAL  NOT NULL default '3 hours',
```
Comment (jpqy) [inline server/db/schema/05_quotas.sql]: Let's lower the default, 3 hours is a lot of time. I suggest 1.5 hours.

**Review by jpqy — CHANGES_REQUESTED** [review]
Reviewed with inline comments.

File: server/routes/userRoutes.js (hunk 1 above)
Comment (riztaha) [reply server/routes/userRoutes.js]: Done

File: server/db/schema/05_quotas.sql (hunk 2 above)
Comment (riztaha) [reply server/db/schema/05_quotas.sql]: Done.

File: server/routes/userRoutes.js (hunk 1 above)
Comment (jpqy) [reply server/routes/userRoutes.js]: You misunderstood, I meant you should do 
```
dbHelper.addQuotaForUser(user.id, "1.5 hours")
.then(()=>res.status(200).json("User created!"))
.catch((err) => res.status(500).json("Invalid Request", err));
```

Every dbHelper call should have a .then and .catch. You can't do something like
```
dbHelper.addQuotaForUser(user.id, "3 hours");
return res.status(200).json("User created!");
```

Comment (riztaha) [reply server/routes/userRoutes.js]: Please review commit 7eeb616

**Comment by riztaha** [comment]
Merged master and removed conflicts.

**Review by jpqy — APPROVED** [review]

**Pull request merged**